docs/Getting_Started.md

```diff
@@ -0,0 +1,421 @@
+
+> warn
+> The sample code uses an object as in-memory storage, but for production apps you should use a database.
+
+When sending a message with [message components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/what-is-a-component), the individual payloads are appended to a `components` array. Actionable components (like buttons) need to be inside of an [action row](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/action-rows), which you can see in the code sample.
+
+Note the unique `custom_id` sent with message components, in this case `accept_button_` with the active game's ID appended to it. A `custom_id` can be used to handle requests that Discord sends you when someone interacts with the component, which you'll see in a moment.
+
+Now when you run the `/challenge` command and pick an option, your app will send a message with an **Accept** button. Let's add code to handle the button press.
+
+### Handling button interactions
+
+When users interact with a message component, Discord will send a request with an [interaction type](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-interaction-type) of `3` (or the  `MESSAGE_COMPONENT` value when using `discord-interactions`).
+
+To set up a handler for the button, we’ll check the `type` of interaction, followed by matching the `custom_id`. Paste the following code under the type handler for `APPLICATION_COMMAND`s:
+
+```javascript
+if (type === InteractionType.MESSAGE_COMPONENT){
+    // custom_id set in payload when sending message component
+    const componentId = data.custom_id;
+
+    if (componentId.startsWith('accept_button_')) {
+        // get the associated game ID
+        const gameId = componentId.replace('accept_button_', '');
+        // Delete message with token in request body
+        const url = DiscordAPI(`webhooks/${process.env.APP_ID}/${req.body.token}/messages/${req.body.message.id}`);
+        try {
+            await res.send({
+                "type": InteractionResponseType.CHANNEL_MESSAGE_WITH_SOURCE,
+                "data": {
+                    // Fetches a random emoji to send from a helper function
+                    "content": "What's your object of choice?",
+                    // Indicates it'll be an ephemeral message
+                    "flags": 64,
+                    "components": [{
+                        "type": ComponentType.ACTION,
+                        "components": [{
+                            "type": ComponentType.SELECT,
+                            // Append game ID
+                            "custom_id": `select_choice_${gameId}`,
+                            "options": getShuffledOptions()
+                        }]
+                    }]
+                }
+            });
+
+            await client({ url, method: 'delete' });
+            return;
+        } catch (err) {
+            console.error(`Error sending message: ${err}`);
```

Comment:
When the error is logged here, will it provide a full callstack?  I can't remember what printing an err like this does exactly 😆

Reply:
it will not. i would recommend `console.error('Error sending message:', err)` instead.